From b9653c08896fcb08c441177e5918458f09111351 Mon Sep 17 00:00:00 2001
From: "Yixun Lan (dlan)" <dennis.yxun@gmail.com>
Date: Sat, 14 Jun 2014 23:16:36 +0800
Subject: [PATCH] fix undefined va_{start,end} error

https://bugs.gentoo.org/show_bug.cgi?id=513196

---

x86_64-pc-linux-gnu-gcc -Wl,-O1 -Wl,--as-needed -rdynamic -pie -o automount auto
mount.o indirect.o direct.o spawn.o module.o mount.o lookup.o state.o flag.o ../
lib/autofs.a -lpthread -lnsl -ldl
../lib/autofs.a(defaults.o): In function `message':
/var/tmp/portage/net-fs/autofs-5.1.0/work/autofs-5.1.0/lib/defaults.c:179: undef
ined reference to `va_start'
/var/tmp/portage/net-fs/autofs-5.1.0/work/autofs-5.1.0/lib/defaults.c:186: undefined reference to `va_end'
collect2: error: ld returned 1 exit status
Makefile:33: recipe for target 'automount' failed
make[1]: *** [automount] Error 1
make[1]: Leaving directory '/var/tmp/portage/net-fs/autofs-5.1.0/work/autofs-5.1.0/daemon'
Makefile:14: recipe for target 'daemon' failed
make: *** [daemon] Error 2

Signed-off-by: Yixun Lan (dlan) <dennis.yxun@gmail.com>
---
 lib/defaults.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/defaults.c b/lib/defaults.c
index 2b03ea2..5728e67 100644
--- a/lib/defaults.c
+++ b/lib/defaults.c
@@ -16,6 +16,7 @@
 
 #include <stdlib.h>
 #include <stdio.h>
+#include <stdarg.h>
 #include <ctype.h>
 #include <string.h>
 #include <sys/utsname.h>
-- 
1.8.5.2

